.golangci.yml

```diff
@@ -29,6 +29,7 @@ linters:
     - typecheck
     - unconvert
     - unused
+    - errcheck
```

Comment:
you should remove this lol.

Reply:
found a couple of places in which because we are not checking errors there were some that were being ignored. I think I solved the most important ones, I leave some of the integration tests for later